openapi.yml

```diff
@@ -896,8 +896,18 @@ components:
               # description: An alphabet or other notation used to represent a
               #   language or other symbolic system of the descriptive element value.
     DOI:
-      type: string
       description: Digital Object Identifier (https://www.doi.org)
+      oneOf:
+        - $ref: '#/components/schemas/DOI_pattern'
+        - $ref: '#/components/schemas/DOI_exceptions'
+    DOI_exceptions:
```

Comment:
I think this will have to be DoiExceptions, so that zeitwork can predict where this file is.

Reply:
yep - got that error and fixed it.